redis/src/main/scala/com/avsystem/commons/redis/config/config.scala

```diff
@@ -169,6 +170,7 @@ case class ConnectionConfig(
   connectTimeout: OptArg[FiniteDuration] = OptArg.Empty,
   idleTimeout: OptArg[FiniteDuration] = OptArg.Empty,
   maxWriteSizeHint: OptArg[Int] = 50000,
+  initTimeout: FiniteDuration = 5.seconds,
```

Comment:
How about `initResponseTimeout`, it would explain this setting better

Also, I think 5 seconds default value is too strict for a library, I would vote for at least 15.